src/components/Menu.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+function Menu() {
```

Comment:
Named this component as `Menu` instead of Feed as I believe that that there will likely be more (horizontal) menu items later.

Reply:
I think this too generic of a name since other pages will have menus and this is not just a menu. This makes it less reusable, why don't you name it `FeedMenu`, `FeedSelector` or something similar.

Reply:
Addressed. The component name has been changed to `FeedMenu`. Thanks!